### PR TITLE
Show anchors for headings

### PR DESCRIPTION
**Pull request by pascalberger**
This PR adds anchor links to all headings (h2 - h6) like eg. in Markdown rendering on GitHub to the default theme. Uses [AnchorJS](https://github.com/bryanbraun/anchorjs).

### CODE REVIEW

File: src/docfx.website.themes/default/styles/docfx.js [hunk 1]
```diff
@@ -20,6 +20,15 @@ $(function () {
     $('.IMPORTANT, .CAUTION').addClass('alert alert-danger');
   })();  
 
+  // Enable anchors for headings.
+  (function () {
+    anchors.options = {
+      placement: 'left',
+      visible: 'touch'
+    };
+    anchors.add('h2, h3, h4, h5, h6');
```
Comment (vicancy) [inline src/docfx.website.themes/default/styles/docfx.js]: add more constraints, e.g. only headers inside `.article` ?

Comment (pascalberger) [inline src/docfx.website.themes/default/styles/docfx.js]: Good point. Updated selector

File: src/docfx.website.themes/default/styles/docfx.js [hunk 2]
```diff
@@ -20,6 +20,15 @@ $(function () {
     $('.IMPORTANT, .CAUTION').addClass('alert alert-danger');
   })();  
 
+  // Enable anchors for headings.
+  (function () {
+    anchors.options = {
+      placement: 'left',
+      visible: 'touch'
+    };
+    anchors.add('article > h2, h3, h4, h5, h6');
```
Comment (vicancy) [inline src/docfx.website.themes/default/styles/docfx.js]: `article` only takes effect on `h2`?

Comment (pascalberger) [inline src/docfx.website.themes/default/styles/docfx.js]: @vicancy Oops, that was definitely done too late at night :) Should now be OK

**Pull request merged**